lib/bundle/brewfile.rb

```diff
@@ -2,22 +2,22 @@
 
 module Bundle
   module Brewfile
+    ENV_KEY = "HOMEBREW_BUNDLE_FILE"
     module_function
 
     def path(dash_writes_to_stdout: false)
+      env_bundle_file = ENV.fetch(ENV_KEY, "")
```

Comment:
Use `ENV["HOMEBREW_BUNDLE_FILE"]`

Reply:
This postpones distinguishing between `nil` and `""` (empty string). I'm using `.fetch` to ensure that subsequent access to `env_bundle_file` is guaranteed to be operating on `String`.